tests/test_gec_model.py

```diff
@@ -0,0 +1,63 @@
+from pathlib import Path
+import torch
+import requests
+from tqdm import tqdm
+
+from allennlp.common.testing import ModelTestCase
+from allennlp.modules.text_field_embedders import BasicTextFieldEmbedder
+from allennlp.data.vocabulary import Vocabulary
+from allennlp.data import Token
+from allennlp.data.instance import Instance
+from allennlp.data.fields import TextField
+from allennlp.data.dataset import Batch
+
+from gector.gec_model import GecBERTModel
+from gector.bert_token_embedder import PretrainedBertEmbedder
+from gector.tokenizer_indexer import PretrainedBertIndexer
+
+
+class TestGecModel(ModelTestCase):
+    """Test class for GecModel"""
+    def setUp(self):
+        super().setUp()
+        self.vocab_path = "data/output_vocabulary"
+        self.vocab = Vocabulary.from_files(self.vocab_path)
+        self.model_name = "roberta-base"
+        model_url = "https://grammarly-nlp-data-public.s3.amazonaws.com/gector/roberta_1_gectorv2.th"
+        test_fixtures_dir_path = Path(__file__).parent.parent / "test_fixtures"
+        model_path = test_fixtures_dir_path / "roberta_1_gectorv2.th"
+        if not model_path.exists():
```

Comment:
This is a bit hacky. I'm downloading the roberta model weights if they don't exist in test_fixtures. 
Once we use an allennlp Predictor, the model archive can be uploded to a public s3 bucket and then the test can just point at the s3 uri.
Until then, this is a way to get predictions from an existing model. If anyone has a better way to do this in mind, I'm totally open to it.

Reply:
Seems reasonable to me as a temporary measure.